sdk/canary/ci.yml

```diff
@@ -10,12 +10,22 @@ trigger:
     include:
     - sdk/canary/
 
+parameters:
+- name: release_azure_canary_core
+  displayName: 'azure_canary_core'
```

Comment:
Nit: again, no quotes needed. Less to type if we show a good example.

Reply:
...though, it does line the crate name up nicely with that extra leading character. Not necessary, but I like the benefit, so I'll resolve this but retain it for posterity.